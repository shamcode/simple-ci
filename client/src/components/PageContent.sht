{% import WithoutFlushLoader from 'sham-ui-without-flush-loader' %}
{% import PageDataLoading from './PageDataLoading.sht' %}

{% defblock breadcrumbs %}

{% if dataLoaded %}
    {% defblock content %}
{% else %}
    <WithoutFlushLoader loadingComponent={{PageDataLoading}}/>
{% endif %}