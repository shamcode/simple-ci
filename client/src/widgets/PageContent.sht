{% import FlushPageDataLoading from './FlushPageDataLoading' %}

{% defblock 'breadcrumbs' %}

{% if dataLoaded %}
    {% defblock 'content' %}
{% else %}
    <FlushPageDataLoading/>
{% endif %}