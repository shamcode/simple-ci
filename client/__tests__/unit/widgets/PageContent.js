import PageContent  from '../../../src/widgets/PageContent.sht';
import renderer from 'sham-ui-test-helpers';

it( 'renders correctly', () => {
    const meta = renderer( PageContent, {} );
    expect( meta.toJSON() ).toMatchSnapshot();
} );
