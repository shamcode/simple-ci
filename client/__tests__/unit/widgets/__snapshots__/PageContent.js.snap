// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Object {
  "Constructor": "PageContent",
  "Options": Object {
    "ID": "widget",
    "containerSelector": "body",
    "types": Array [],
  },
  "html": "
<!--breadcrumbs-->
<!--if-->
",
}
`;
